src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,117 @@
 package core.basesyntax;
 
+import java.util.Objects;
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int DEFAULT_CAPACITY = 16;
+    private static final int CAPACITY_INCREASE = 2;
+    private static final float LOAD_FACTOR = 0.75f;
+    private Node<K,V>[] table = new Node[DEFAULT_CAPACITY];
+    private int size;
 
     @Override
     public void put(K key, V value) {
-
+        checkThreshold();
+        int bucketIndex = key == null ? 0 : getBucketIndex(hash(key));
```

Comment:
if you have `getBucketIndex` method it must give you ready for use index
```suggestion
        int bucketIndex = getBucketIndex(key);
```